.github/workflows/release-gh-notes.yml

```diff
@@ -11,6 +11,10 @@ on:
         required: true
         description: 'npm package of the release repo'
         type: string
+      commit:
+        required: false
```

Comment:
Why is it now required?

Reply:
cause the main PR is still WIP: https://github.com/cloudscape-design/actions/pull/5

Reply:
Yes, I want to make it backward compatible, so we cannot turn the new process on and off without opening too many PRs